src/Pub/Broadcasting/Broadcasters/EventBridgeBroadcaster.php

```diff
@@ -44,15 +44,15 @@ public function broadcast(array $channels, $event, array $payload = []): void
     /**
      * @inheritDoc
      */
-    public function auth($request)
+    public function auth($request): bool
     {
         return true;
```

Comment:
In https://github.com/Pod-Point/laravel-aws-pubsub/pull/38/files we don't return true here or in the method below